### PR TITLE
Handle event click categories, detail, home screen

### PR DESCRIPTION
**Pull request by chicky-dev**
https://edu-redmine.framgia.vn/issues/3567

### CODE REVIEW

**Comment by chicky-dev** [comment]
ready

File: app/src/main/java/com/framgia/vhlee/themoviedb/ui/detail/VideoFragment.java [hunk 1]
```diff
@@ -55,11 +56,16 @@ public void onInitializationFailure(YouTubePlayer.Provider provider,
     public void setVideoId(String videoId, YouTubePlayer.OnFullscreenListener listener) {
         mListener = listener;
         mVideoId = videoId;
+        Log.d(TAG, "setVideoId: " + videoId);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/vhlee/themoviedb/ui/detail/VideoFragment.java]: remove log

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by VoLy** [comment]
ready

**Comment by chicky-dev** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**